drivers/ethernet/eth_sam_gmac.c

```diff
@@ -1792,9 +1792,14 @@ static void get_mac_addr_from_i2c_eeprom(uint8_t mac_addr[6])
 		return;
 	}
 
-	i2c_write_read(dev, CONFIG_ETH_SAM_GMAC_MAC_I2C_SLAVE_ADDRESS,
-		       &iaddr, CONFIG_ETH_SAM_GMAC_MAC_I2C_INT_ADDRESS_SIZE,
-		       mac_addr, 6);
+	int ret = i2c_write_read(dev, CONFIG_ETH_SAM_GMAC_MAC_I2C_SLAVE_ADDRESS,
```

Comment:
We try to introduce variables at the beginning of the block, so could you place the `int ret;` at the beginning of the function. Other than that, LGTM

Reply:
Fixed 😄

Reply:
@jukkar I wasn't sure if the order of the variables at the top of the function mattered. But I put `int ret` below `uint32_t`. I vaguely remember seeing this being an issue with some standard like MISRA but not sure.